rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] Hello there! 🤗 Please make sure to include a setter for the @book and @person variables. This is important because we'll need to not only read but also make modifications to it. 📝✏️ By adding a setter, we can ensure that the @book and @person variables become moree flexible and adaptable, allowing us to make the necessary changes as required. 🔧💼

```suggestion
  attr_accessor :date, :book, :person
```